ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -62,11 +62,10 @@
                 <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">
                     Notifying TB service (required)
                 </label>
-                <span class="nhsuk-error-message" id="tb-service-error"
+                <span nhs-span-type="ErrorMessage" id="tb-service-error" has-error="@TBServiceCodeError"
                       ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
                 <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode"

```

Comment:
Did we decide in scoping whether this should default to a label if only one tb service was available?

Reply:
I think you're thinking of https://airelogic-nis.atlassian.net/browse/NTBS-210? Having it as a label when it's not editable

Reply:
I think a question was raised whether when there's only 1 entry in the dropdown to bother with the dropdown - but I recall now that that option doesn't allow for unsetting... so I think we dismissed it.

Reply:
oh yeah, that sounds familiar. but agreed, we should just keep it as dropdown